src/Explorer/Explorer.tsx

```diff
@@ -1729,7 +1729,7 @@ export default class Explorer {
     }
 
     const databaseAccount = this.databaseAccount();
-    const databaseAccountLocation = databaseAccount && databaseAccount.location.toLowerCase();
+    const firstWriteLocation = databaseAccount?.properties?.writeLocations[0].locationName.toLowerCase();
```

Comment:
Do we need to check if `writeLocations` is defined and contains at least 1 element?

Reply:
Fixing it in the next iterations